xml/virt_logs.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-logs">
+ <title>Gathing system information and logs</title>
```

Comment:
I would separate these two.
Users might be interested in collecting logs independently from a troubleshooting session with SUSE.

So I'd have <title>Gathering System Information for Virtualization Host troubleshooting</tile>
where we  would talk about supportconfig, mentioning that often collecting from both host and guest can be useful when possible etc, and then for the logging mentioning to also include libvirt log, and the libvirt driver log, and make a reference to the following section.

Reply:
maybe it already is, and I am confused about the tags.

Reply:
The supportconfig tool already grabs everything we need. If it is missing something, we fix it. IMO it is sufficient to mention the tool as a way to collect all needed info. For tougher bugs, we may need to tweak the system and collect additional info. The "logs" part is about that.